databricks_cli/unity_catalog/uc_service.py

```diff
@@ -0,0 +1,448 @@
+# Databricks CLI
```

Comment:
Can the uc service be autogenerated from proto file ([ref link](https://databricks.atlassian.net/wiki/spaces/UN/pages/439517219/Contributing+to+Databricks+CLI))?

If it can be auto-generated, then the class will exist in `sdk/service.py` as well and automate manual efforts for syncing REST API changes to the CLI.

Reply:
Ah i just saw your comment that it's temporary. It's okay then.